docs/osoc.yaml

```diff
@@ -184,7 +184,7 @@ paths:
                     name: view
                     schema:
                         $ref: '#/components/schemas/StudentView'
-                    description: The view of the student, with different views the students will contain less or more information (default is Full). The Basic view can be used to get basic information of the student to display it in a listing, this way unnecessary data does not need to be sent.
+                    description: The view of the student, with different views the students will contain less or more information (default is Full). The List view can be used to get basic information of the student to display it in a listing, the Communication view can be used to just get info about the student and his/hers communication and the Basic view can be used to show the bare minimum info of a student. This way unnecessary data does not need to be sent.
```

Comment:
Could you put this on multiple lines? Also this doesn't really explain the difference between the List and Basic views.